front/src/edittracks/tracks.js

```diff
@@ -238,14 +236,7 @@ const TrackCRUD = () => {
 
   const partialFetch = lodash.partial(fetch, '/api/track'); // eslint-disable-line
 
-  const token = localStorage.getItem('token');
-
-  const opts = {
-    headers: {
-      Authorization: `Bearer ${token}`,
-    },
-  };
```

Comment:
opts has to be removed from rows 137, 192 ( not defined  no-undef )

Reply:
Removed.